src/components/MainPanel/ModePicker.scss

```
@@ -0,0 +1,88 @@
+// Segmented control. Color palette and structure mirror the
+// .segmented-control.noise-suppression-modes in Settings.scss so the two
+// segmented controls look like the same component across the app:
+//   - 1px outer border (var --border-color, #333)
+//   - Each option fills with var --bg-secondary (#1a1a1a)
+//   - Options separated by border-right (not a gap)
+//   - Hover var --bg-hover (#252525)
+//   - Active #10a37f / #fff
+//
```

Stylelint (17.11.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)